src/en-GB/solutions/case-studies/README.md

```diff
@@ -0,0 +1,140 @@
+<p align="center"><img src="https://i2.wp.com/ceph.io/wp-content/uploads/2016/07/Ceph_Logo_Standard_RGB_120411_fa.png?resize=322%2C148&ssl=1" alt="Ceph" /></p>
+
+# Case studies
+
+## Page data
+
+At the beginning of all case studies, there's a block of data wrapped with ---. This is called frontmatter, and defines data specific to that page in the site.
+
+The frontmatter data is written in [YAML format](https://yaml.org/) using `key: value` pairs. Frontmatter for the case study pages follows a specific schema:
+
+```yaml
+---
+title: Case study title
+image: "/assets/image.jpg"
+sponsor:
+  name: name
+  logo: "/assets/logo.png"
+  website: "https://www.website.com"
+tags:
+  - taxonomy
+  - taxonomy
+---
+
+```
+
+- `title` (String) — Determines the link text as it appears in the case study navigation card. Also used as the `<title>` for the page.\*
+- `image` (Asset path) — Determines the image shown in the case study navigation card and the hero image beneath the title on the case study page.
```

Comment:
Wondering whether the types e.g. (`String`) here are always the _type_-type (String, Boolean, Integer, Array (or List even)), and we describe more of what the Strings are doing in their description. Spitballin'

```
- `image` (String) - Path to asset file. Determines the image shown in the case study navigation card and the hero image beneath the title on the case study page.
```

Whaddya think?

Reply:
Yeah, seems more accurate. 

Done.